### ISSUE TITLE
Problem with cleanjoke pk3

### ISSUE THREAD

**Issue by Yberion** [Razish/cleanjoke#10]
When generating cleanjoke pk3 the hierarchy is wrong, it contains subfolders when it shouldn't.

**Comment by Razish** [comment]
Do you mean `JediAcademy/` root folder? Due to OJK supporting JO+JA builds in separate dirs

**Comment by Yberion** [comment]
No, it was like nearly full path in the .pk3 starting from the building folder. I can't make it to pack on my main computer, no idea why.
